cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -289,21 +289,34 @@ def _add_head_node_instance_metrics_graphs(self):
         # Create a text widget for subtitle "Head Node Instance Metrics"
         self._add_text_widget("## Head Node Instance Metrics")
 
-        # EC2 metrics graph for head node instance
+        # EC2 metrics for graph for head node instance
         ec2_metrics = [
-            new_pcluster_metric(title="CPU Utilization", metrics=["CPUUtilization"]),
-            new_pcluster_metric(title="Network Packets In/Out", metrics=["NetworkPacketsIn", "NetworkPacketsOut"]),
-            new_pcluster_metric(title="Network In and Out", metrics=["NetworkIn", "NetworkOut"]),
-            new_pcluster_metric(title="Disk Read/Write Bytes", metrics=["DiskReadBytes", "DiskWriteBytes"]),
-            new_pcluster_metric(title="Disk Read/Write Ops", metrics=["DiskReadOps", "DiskWriteOps"]),
+            new_pcluster_metric(title="CPU Utilization", metrics=["CPUUtilization"], namespace="AWS/EC2"),
+            new_pcluster_metric(
+                title="Network Packets In/Out", metrics=["NetworkPacketsIn", "NetworkPacketsOut"], namespace="AWS/EC2"
+            ),
+            new_pcluster_metric(title="Network In and Out", metrics=["NetworkIn", "NetworkOut"], namespace="AWS/EC2"),
+            new_pcluster_metric(
+                title="Disk Read/Write Bytes", metrics=["DiskReadBytes", "DiskWriteBytes"], namespace="AWS/EC2"
+            ),
+            new_pcluster_metric(
+                title="Disk Read/Write Ops", metrics=["DiskReadOps", "DiskWriteOps"], namespace="AWS/EC2"
+            ),
+        ]
+
+        # CW Agent metrics for graph for head node instance
+        cwagent_metrics = [
+            new_pcluster_metric(title="Disk Used Percent", metrics=["disk_used_percent"], namespace="CWAgent"),
```

Comment:
note: if we decide to change the aggregation metrics for disk_used_percent, we will need to update here accordingly